.circleci/config.yml

```diff
@@ -32,6 +32,7 @@ jobs:
       - save_cache:
           key: 'core-node10-{{ checksum "checksum.txt" }}-1'
           paths:
+            - ./packages/client/node_modules
```

Comment:
This package no longer exists in this repository, please clean up those old files or do a fresh clone.

Reply:
Will do.  I'll get this done tonight.

Reply:
@faustbrian  done